pkg/utils/handler.go

```diff
@@ -0,0 +1,20 @@
+//
+// Copyright (C) 2023 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package utils
+
+import (
+	"net/http"
+
+	"github.com/labstack/echo/v4"
+)
+
+// WrapHandler wraps `handler func(http.ResponseWriter, *http.Request)` into `echo.HandlerFunc`
+func WrapHandler(handler func(http.ResponseWriter, *http.Request)) echo.HandlerFunc {
+	return func(c echo.Context) error {
+		handler(c.Response(), c.Request())
+		return nil
+	}
+}
```

Comment:
Is this something that should be in go-mod-bootstrap utils? 
I.e. need in edgex-go and app sdk also?

Reply:
Change made to move `WrapHandler` to bootstrap in https://github.com/edgexfoundry/go-mod-bootstrap/pull/576